### DISCUSSION TITLE
My feed has garbage

### DISCUSSION THREAD

**Discussion by neobuddy89** [community/community#67300] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

Your description says I can customize my feed as per my liking and provides no option to customize and watch project I follow.

Please do away with this shiny garbage,

All that glitters is not gold.

**Comment by 1atAlcone** [comment]
I think, it has defining to make a save. But in the other way around it was mean to say make am options for something to save it by the need of your own.

**Comment by 1atAlcone** [comment]
It is kinda like an idea that makes you more than what to say I guess. No harm feelings. No offence. 😏
